checker/src/main/java/org/checkerframework/checker/signedness/SignednessUtil.java

```diff
@@ -372,11 +378,11 @@ public static int compareUnsigned(@Unsigned long x, @Unsigned long y) {
     /**
      * Compares two unsigned ints x and y.
      *
-     * <p>This is a reimplementation of Java 8's {@code Integer.compareUnsigned(int, int)}.
```

Comment:
I don't think you need to remove comments like this, which are still helpful.
This pull request can just add the deprecation information.